phoenix-core/src/main/java/org/apache/phoenix/schema/transform/TransformMaintainer.java

```diff
@@ -21,7 +21,6 @@
 import org.apache.phoenix.thirdparty.com.google.common.collect.Maps;
 import org.apache.phoenix.thirdparty.com.google.common.collect.Sets;
 
-import org.apache.hadoop.hbase.client.Delete;
```

Comment:
I can see a lot of files where you are just randomly removing unused imports.
This change is already huge, leaving these alone for now would make it somewhat more managable.

We can handle unrelated cleanups in separate tickets.

This only applies when you only remove non-server unused imports.
When there are other changes, it's fine to clean up them.